libcaf_io/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+# check for BIICode build
+if (BIICODE)
```

Comment:
This line here detects if this make file is being evaluated as a result of running ``bii build`` and then short circuits evaluation to only include biicode specific make targets i.e. allow biicode uses its own auto generated CMakefile.txt